### PR TITLE
Define models and relationships for API

### PR DESCRIPTION
**Pull request by gashcrumb**

### CODE REVIEW

**Comment by gashcrumb** [comment]
@kahboom FYI if you want to take a peek at some point

**Comment by kahboom** [comment]
@gashcrumb - Thanks, this looks amazing! 👍 Only thing I can add is that I think we should make it clearer which are columns / model attributes versus which are just tables / entities, but I can help do that with a diagram if you aren't working on one already.

**Comment by gashcrumb** [comment]
If you take a look at the little blurb at the end of docs/entities.md
there's some info on how to take the entities.xml and load it up in the
little schema design tool I've been using, which hopefully makes it a
little clearer, but yeah, probably worth making that clear in the readme
too.

**Comment by kahboom** [comment]
@gashcrumb Awesome, didn't see that. I may still create an image file with a diagram if I get a chance, which of course is susceptible to becoming out of date frequently, but just so that it's easy to visualize as soon as people see the docs without having to do those steps.

**Comment by gashcrumb** [comment]
@kahboom found a better tool to get a screenshot of the graph, have updated entities.md to include it.

Be kinda cool to generate the sequelize code from the XML this tool generates, but that might be taking it too far :-)

File: src/models/Connection/sequelize.js [hunk 1]
```diff
@@ -12,15 +11,30 @@ module.exports = function(sequelize, DataTypes) {
...
   }, {
     classMethods: {
-      associate: function(models) {}
+      associate: function(models) {
+        const Connection = models['Connection'];
+        const ConnectionType = models['ConnectionType'];
+        const Organization = models['Organization'];
+        const Tag = models['Tag'];
+
+        Connection.belongsTo(Organization);
```
Comment (kahboom) [inline src/models/Connection/sequelize.js]: You learned this super quickly! Awesome.

Comment (gashcrumb) [inline src/models/Connection/sequelize.js]: @kahboom lol, yeah, the docs are really good so that helps!

**Pull request merged**